pkg/storaged/dialog.jsx

```diff
@@ -360,6 +360,7 @@ export const dialog_open = (def) => {
         return {
             id: "dialog",
             title,
+            variant: fields.some(f => f.large_modal) ? "large" : null,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)